projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -287,12 +294,15 @@ class_exists( 'Jetpack_AMP_Support' )
 	 * Check if the content of a post uses gallery blocks. To be used by 'the_content' filter.
 	 *
 	 * @since 6.8.0
+	 * @deprecated $$next-release$$ We now hook into the 'block_render_{block_name}' hook to add markup.
```

Comment:
```suggestion
	 * @deprecated $$next-version$$ We now hook into the 'block_render_{block_name}' hook to add markup.
```

Reply:
This was 100% due to me mis-remembering the specific syntax -- I correctly remembered most of it but clearly got the `version` part of it wrong! I've fixed the usages and injected it in the `_deprecated_function()` call as well.